app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ProfileController < ApplicationController
+  before_filter :authenticate_user!, only: [ :create, :update, :destroy ]
+  respond_to :json
+  def index
+    #return recipes's current_user.
```

Comment:
we have before_filter : authentificate_user!

2016-01-25 11:08 GMT+01:00 Sami <notifications@github.com>:

> In app/controllers/profile_controller.rb
> <https://github.com/melas123/CookingRecipe/pull/29#discussion_r50674368>:
>
> > @@ -0,0 +1,10 @@
> > +class ProfileController < ApplicationController
> > +  before_filter :authenticate_user!, only: [ :create, :update, :destroy ]
> > +  respond_to :json
> > +  def index
> > +    #return recipes's current_user.
>
> before you return recipe for the current user you must test if user signed
> in :
> if user_signed_in?
> #return user recipes
> , and use named scope
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/melas123/CookingRecipe/pull/29/files#r50674368>.
>